site/sass/components/_card-study.scss

```
@@ -0,0 +1,162 @@
+@use 'sass:math';
+
+.card-study {
+  $break-2-col: 980px;
+  $break-feature-col: 690px;
+  $break-featured-stack: 0 689px;
+  $scale-max: 1600px;
+  $clamp-check: clamp(1px, 1vw, 2px);
+
+  border-radius: 0.625rem;
+  display: grid;
+  grid-template-rows: min-content;
+  overflow: hidden;
+  position: relative;
+
+  @include mq($break-2-col) {
+    grid-template: auto / 41.5% 1fr;
+  }
+
+  &,
+  &__tile {
+    background-color: var(--white);
+  }
+
+  &__media {
+    aspect-ratio: 2 / 1;
+    height: 100%;
+    object-fit: cover;
+  }
+
+  &__tile {
+    align-items: flex-start;
+    display: flex;
+    flex-direction: column;
+    padding: 2rem 1.5rem;
+
+    @include mq(0 419px) {
+      align-items: center;
+      text-align: center;
+    }
```

```suggestion
    text-align: center;
    align-items: center
    @include mq(419px) {
      align-items: revert;
      text-align: revert;
    }
```

Instead of clamping from 0 to a max-width like this, or using a `max-width` breakpoint, invert the styling so you only need a `min-width`. This ensures that our single column layout works without breakpoints, which should be our "mobile first" development goal as much as possible